### ISSUE TITLE
neovim; CTRL-^ not working.

### ISSUE THREAD

**Issue by molleweide** [kmonad/kmonad#260]
(this is not super urgent).

Anyone else noticing that comands such as `CTRL-^` for switching to alternate file
not working in vim (neovim).

I can see in vim that it registers only `6` and I am not using shift-hold to generate `^`, so I am not sure what's going on really. I have `lctl` in my base layer and then reach the `^` in another layer via homerow tapnextreleasehold.
I haven't done a deep dive on the issue yet but I know for sure that it did work maybe two weeks ago I believe. And I know that I have pulled eb6f1ef from upstream couple of days ago, but I am not sure if the issue arose with the merge.

**Comment by thoelze1** [comment]
Are you sure it's possible to send <kbd>CONTROL</kbd><kbd>SHIFT</kbd><kbd>6</kbd> to neovim at all, even while kmonad isn't running?

**Comment by molleweide** [comment]
Yes ofc i always use it before

**Comment by molleweide** [comment]
Oohh!! I remeber also that it stopped working once, and then i restarted kmonad, BUT NOW restarting does not rectify proble

**Comment by thoelze1** [comment]
If this is still happening, post your config and some output from running kmonad with `-l debug`

**Comment by molleweide** [comment]
> If this is still happening, post your config and some output from running kmonad with `-l debug`

Yes! I'll do this later. I have noticed that this problem persists many shifted symbols when they are located in a second hold-layer.
